lib/octocore/helpers/api_consumer_helper.rb

```diff
@@ -22,7 +22,8 @@ def api_events
 
       def handle(msg)
           msg_dump = msg
```

Comment:
Please remove this line as it is no longer needed. It's references later can be replaced by an equivalent `msg_obh.message` or similar call.